queries/angular/highlights.scm

```diff
@@ -21,47 +42,67 @@
   function: ((identifier) @function.builtin
     (#eq? @function.builtin "$any")))
 
+(pair
+  key: ((identifier) @variable.builtin
+  (#eq? @variable.builtin "$implicit")))
+
+(control_keyword) @type.control
+(special_keyword) @type
```

Comment:
What are these two?

Reply:
Control keywords are the `if/else/switch/defer/for` etc keywords. 

Special keywords are keywords used in aliases or other expressions. E.g. `let items of items` where `let` and `of` are special. 
Or `@placeholder(minimum 1s)` where `minimum` is a special keyword.

Reply:
Then those highlights are wrong. See CONTRIBUTING.md.

Reply:
"if" -> `@conditional`
"for" -> `@repeat`
etc.

Reply:
Ah, yes. I would have to modify the parser a bit then.

Reply:
Not really, just use `#any-of`.